sympy/physics/continuum_mechanics/arch.py

```diff
@@ -0,0 +1,155 @@
+from sympy import sympify, symbols, solve, Symbol
+
+class Arches:
+    """
+    Example
+    ========
+    >>> from sympy.physics.continuum_mechanics.arch import Arches
+    >>> a = Arches((0,0),(10,0),crown_x=5,crown_y=5)
+    >>> a.get_parabola_eqn
+    5 - (x - 5)**2/5
+
+    >>> from sympy.physics.continuum_mechanics.arch import Arches
+    >>> a = Arches((0,0),(10,1),crown_x=6)
+    >>> a.get_parabola_eqn
+    9/5 - (x - 6)**2/20
+    """
+    def __init__(self,left_support,right_support,**kwargs):
+        self._shape_eqn = None
+        self._left_support  = (sympify(left_support[0]),sympify(left_support[1]))
+        self._right_support  = (sympify(right_support[0]),sympify(right_support[1]))
+        self._crown_x = None
+        self._crown_y = None
+        if 'crown_x' in kwargs:
+            self._crown_x = sympify(kwargs['crown_x'])
+        if 'crown_y' in kwargs:
+            self._crown_y = sympify(kwargs['crown_y'])
+        self.get_parabola_eqn
+        self._loads = {}
+        self._conc_loads = {}
+        self._distributed_loads = {}
+        self._supports = {'left':None,'right':None}
+        self._rope = None
+        self._reaction_force = {Symbol('R_A_x'):0, Symbol('R_A_y'):0, Symbol('R_B_x'):0, Symbol('R_B_y'):0}
+        # self._crown = (sympify(crown[0]),sympify(crown[1]))
+
+    @property
+    def get_parabola_eqn(self):
+        "returns the equation of the shape of arch developed"
+        if self._shape_eqn:
+            return self._shape_eqn
+
+        x,y,c = symbols('x y c')
+        a = Symbol('a',positive=False)
+        if self._crown_x and self._crown_y:
+            x0  = self._crown_x
+            y0 = self._crown_y
+            parabola_eqn = a*(x-x0)**2 + y0 - y
+            eq1 = parabola_eqn.subs({x:self._left_support[0], y:self._left_support[1]})
+            solution = solve((eq1),(a))
+            parabola_eqn = solution[0]*(x-x0)**2 + y0
+            if(parabola_eqn.subs({x:self._right_support[0]}) != self._right_support[1]):
+                raise ValueError("provided coordinates of crown and supports are not consistent with parabolic arch")
+            self._shape_eqn = parabola_eqn
+
+        elif self._crown_x:
```

Comment:
If I try to create an `Arch` object with `crown_x=5` and `crown_y=5` and the supports `(0, 0)` and `(10, 0)`, I get no problem, but if I try to create an object with the same supports and only `crown_x=5`, it gives me a `ValueError`

```
>>> from sympy.physics.continuum_mechanics.arch import Arches
>>> a = Arches((0,0),(10,0),crown_x=5, crown_y=5)
>>> a.get_parabola_eqn
5 - (x - 5)**2/5
>>> a = Arches((0,0),(10,0),crown_x=5)
    .
    .
    .
    raise ValueError("parabolic arch cannot be constructed with the provided coordinates")
ValueError: parabolic arch cannot be constructed with the provided coordinates

```

Reply:
That happens cause when we use crown_x = 5 , all values of crown_y are possible i.e. we will get multiple solutions to the parabolic equation.

Reply:
Go and change the Value error then. Will resolve the conversation then.